brave/brave-core
# Fixes date for recurring banner

Resolves https://github.com/brave/brave-browser/issues/3481
Resolves https://github.com/brave/brave-browser/issues/3494

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
Plan 1
- clean profile
- enable rewards
- claim grant
- go to a site
- do recurring tip
- make sure that date is correct
- open panel again on the same page
- do a tip
- make sure that you see one time tip thank you page 

## Reviewer Checklist:
